LICENSE

```diff
@@ -0,0 +1,9 @@
+Copyright © 2016 Catalin Luntraru
```

Comment:
From https://redux-minimal.js.org/ ???  Can you throw something about that into the README -- she has good "get up and running" on her front page :-)